hdl21/primitives.py

```diff
@@ -640,6 +667,8 @@ class BipolarParams:
     tp = Param(
         dtype=BipolarType, desc="Bipolar Type (NPN/ PNP)", default=BipolarType.NPN
     )
+    model = Param(dtype=Optional[str], desc="Model (Name)", default=None)
```

Comment:
Good additions. Noting these generic BJTs have not gotten much use to date, and could use some TLC!